leopard.go

```
@@ -1171,6 +1265,30 @@ func initMul16LUT() {
+}
+
+func (r *leopardFF16) getWorkSlice(n int) [][]byte {
+	if s, ok := r.workSlicePool.Get().([][]byte); ok && cap(s) >= n {
+		return s[:n]
+	}
+	return make([][]byte, n)
+}
+
+func (r *leopardFF16) putWorkSlice(s [][]byte) {
+	clear(s)
+	r.workSlicePool.Put(s)
```

golangci-lint (2.11.4)

[error] SA6002: argument should be pointer-like to avoid allocations

(staticcheck)